spec/mongoid/tasks/database_rake_spec.rb

```diff
@@ -353,7 +353,7 @@
 
     expect_any_instance_of(Mongo::ClientEncryption)
       .to receive(:create_data_key)
-      .with('local')
+      .with('local', {})
```

Comment:
This change is also not immediately related to this issue, but was causing the auto encryption specs to fail because the method was being invoked with an empty hash, causing the expectation to fail.